profiles/NIST_SP-800-53_rev4_MODERATE-baseline_profile.json

```diff
@@ -2566,7 +2566,7 @@
           "description": "NIST Special Publication 800-53 Revision 4: Security and Privacy Controls for Federal Information Systems and Organizations",
           "rlinks": [
             {
-              "href": "https://raw.githubusercontent.com/usnistgov/oscal-content/main/nist.gov/SP800-53/rev4/json/NIST_SP-800-53_rev4_catalog.xml",
+              "href": "NIST_SP-800-53_rev4_MODERATE-baseline_profile.json",
```

Comment:
This may need to be a full URL. I am curious to see what happens in the editor with this.